app/assets/javascripts/govuk_publishing_components/components/accordion.js

```diff
@@ -1,5 +1,282 @@
-// This component relies on JavaScript from GOV.UK Frontend
```

Comment:
Not a criticism but a note: We need to remember to revert this back to a reference to the govuk frontend script once the design system team eventually adopt our accordion, as has been agreed. Is it worth adding a comment here to this effect?

Reply:
🤔 Good point.  I'm think as this will be a piece of work to convert the gem back to this Design System I'm thinking this will be implied from other components use.   I'm not sure about a `TODO` of this nature with this, might be better placed within a card?

Reply:
[Could we raise it as an issue?](https://github.com/alphagov/govuk_publishing_components/issues)

Reply:
Sound good, will do.

Reply:
Added: https://github.com/alphagov/govuk_publishing_components/issues/1897